### PR TITLE
Adding 8ball command

### PR DESCRIPTION
**Pull request by ModoSN**
Magic 8 ball?

### CODE REVIEW

File: commands/other/8ball.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          prompt: 'What do you want to ask?',
+          type: 'string'
+        }
+      ]
+    });
+  }
+
+  run(message) {
+        const ballanswers = fs.readFileSync( 
+      'resources/fun/8ball.json',
```
Comment (Bacon-Fixation) [inline commands/other/8ball.js:23]: 'resources/other/8ball.json',
you have it in the other folder in you repo

**Review by Bacon-Fixation — COMMENTED** [review]
thought it would be fun to try and found this

File: commands/other/8ball.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class EightBallCommand extends Command {
+  constructor(client) {
+    super(client, {
+      name: '8ball',
+      aliases: ['eightball'],
+      memberName: '8Ball',
```
Comment (Bacon-Fixation) [inline commands/other/8ball.js:8]: i get and errors that memberNames need to be lowercase  suggest memberName: '8ball'

**Review by galnir — APPROVED** [review]

File: commands/other/8ball.js (hunk 2 above)
Comment (galnir) [reply commands/other/8ball.js:8]: yeah you're right I missed that

File: commands/other/8ball.js (hunk 1 above)
Comment (Bacon-Fixation) [inline commands/other/8ball.js:23]: location doesn't exist  maybe 'resources/other/8ball.json like in your pr

Comment (galnir) [reply commands/other/8ball.js]: spot on again

**Pull request merged**